api/v1/controllers/userTokens.js

```diff
@@ -25,10 +25,10 @@ const authUtils = require('../helpers/authUtils');
  */
 function whereClauseForUser(userNameOrId) {
   const whr = {};
-  if (u.looksLikeId()) {
+  if (u.looksLikeId(userNameOrId)) {
     // need to use '$table.field$' for association fields
     whr.where = { '$User.id$': {} };
-    whr.where['$User.id$'][cnstnts.SEQ_LIKE] = userNameOrId;
+    whr.where['$User.id$'] = userNameOrId;
```

Comment:
Doing it this way instead of using `$ilike` means that we would no longer return a record with id `6da84a2a-f4b7-4ff1-b97b-56ce92501361` if we put `6da84a2a-F4B7-4ff1-b97b-56ce92501361` in the url because of uppercase/lowercase mismatch. I'm OK with that, and it's better for db performance, but then we should document that record ids in a url are case sensitive.